File: sdkwork-backend-sdk-go/types/plus_disk_member_form.go
package types

// Disk member creation form
type PlusDiskMemberForm struct {
	DiskId int `json:"diskId"`
	UserId int `json:"userId"`
	Permission DiskPermission `json:"permission"`
	Remark string `json:"remark"`
	IsOwner bool `json:"isOwner"`
}

File: sdkwork-backend-sdk-typescript/src/types/plus-disk-member-form.ts
import type { DiskPermission } from './disk-permission';

/** Disk member creation form */
export interface PlusDiskMemberForm {
  /** Disk ID */
  diskId: number;
  /** User ID */
  userId: number;
  /** Disk permission */
  permission: DiskPermission;
  /** Remark */
  remark?: string;
  /** Whether the member is the owner */
  isOwner?: boolean;
}
